File: model/conductor/Transportista.java
/**
 * 
 */
package org.sieca.everis.pdcc.model.conductor;

/**
 * @author lmoremos
 *
 */
public class Transportista {

	
	private String CODIGO_TRANSPORTISTA;
	private String TIPO_DOCUMENTO;
	private String NUMERO_IDENTIFICACION;
	private String RAZON_SOCIAL;
	/**
	 * 
	 */
	public Transportista() {
		super();
	}
	/**
	 * @param CODIGO_TRANSPORTISTA
	 * @param TIPO_DOCUMENTO
	 * @param NUMERO_IDENTIFICACION
	 * @param RAZON_SOCIAL
	 */
	public Transportista(String CODIGO_TRANSPORTISTA, String TIPO_DOCUMENTO, String NUMERO_IDENTIFICACION,
			String RAZON_SOCIAL) {
		this.CODIGO_TRANSPORTISTA = CODIGO_TRANSPORTISTA;
		this.TIPO_DOCUMENTO = TIPO_DOCUMENTO;
		this.NUMERO_IDENTIFICACION = NUMERO_IDENTIFICACION;
		this.RAZON_SOCIAL = RAZON_SOCIAL;
	}
	/**
	 * @return the CODIGO_TRANSPORTISTA
	 */
	public String getCODIGO_TRANSPORTISTA() {
		return CODIGO_TRANSPORTISTA;
	}
	/**
	 * @param CODIGO_TRANSPORTISTA the CODIGO_TRANSPORTISTA to set
	 */
	public void setCODIGO_TRANSPORTISTA(String CODIGO_TRANSPORTISTA) {
		this.CODIGO_TRANSPORTISTA = CODIGO_TRANSPORTISTA;
	}
	/**
	 * @return the TIPO_DOCUMENTO
	 */
	public String getTIPO_DOCUMENTO() {
		return TIPO_DOCUMENTO;
	}
	/**
	 * @param TIPO_DOCUMENTO the TIPO_DOCUMENTO to set
	 */
	public void setTIPO_DOCUMENTO(String TIPO_DOCUMENTO) {
		this.TIPO_DOCUMENTO = TIPO_DOCUMENTO;
	}
	/**
	 * @return the NUMERO_IDENTIFICACION
	 */
	public String getNUMERO_IDENTIFICACION() {
		return NUMERO_IDENTIFICACION;
	}
	/**
	 * @param NUMERO_IDENTIFICACION the NUMERO_IDENTIFICACION to set
	 */
	public void setNUMERO_IDENTIFICACION(String NUMERO_IDENTIFICACION) {
		this.NUMERO_IDENTIFICACION = NUMERO_IDENTIFICACION;
	}
	/**
	 * @return the RAZON_SOCIAL
	 */
	public String getRAZON_SOCIAL() {
		return RAZON_SOCIAL;
	}
	/**
	 * @param RAZON_SOCIAL the RAZON_SOCIAL to set
	 */
	public void setRAZON_SOCIAL(String RAZON_SOCIAL) {
		this.RAZON_SOCIAL = RAZON_SOCIAL;
	}
	
}

File: model/conductor/Transportista.php
/*/
@Yimis Torreglosa Diaz
/*/
<?php class Transportista
{
    private string $CODIGO_TRANSPORTISTA;
    private string $TIPO_DOCUMENTO;
    private string $NUMERO_IDENTIFICACION;
    private string $RAZON_SOCIAL;

    //Constructor
    public function __construct()
    {
        //
    }

    /**
     * Get the value of CODIGO_TRANSPORTISTA
     */

    public function getCODIGO_TRANSPORTISTA()
    {
        return $this->CODIGO_TRANSPORTISTA;
    }

    /**
     * Set the value of CODIGO_TRANSPORTISTA
     *
     * @return  self
     */

    public function setCODIGO_TRANSPORTISTA($CODIGO_TRANSPORTISTA)
    {
        $this->CODIGO_TRANSPORTISTA = $CODIGO_TRANSPORTISTA;

        return $this;
    }

    /**
     * Get the value of TIPO_DOCUMENTO
     */

    public function getTIPO_DOCUMENTO()
    {
        return $this->TIPO_DOCUMENTO;
    }

    /**
     * Set the value of TIPO_DOCUMENTO
     *
     * @return  self
     */

    public function setTIPO_DOCUMENTO($TIPO_DOCUMENTO)
    {
        $this->TIPO_DOCUMENTO = $TIPO_DOCUMENTO;

        return $this;
    }

    /**
     * Get the value of NUMERO_IDENTIFICACION
     */

    public function getNUMERO_IDENTIFICACION()
    {
        return $this->NUMERO_IDENTIFICACION;
    }

    /**
     * Set the value of NUMERO_IDENTIFICACION
     *
     * @return  self
     */

    public function setNUMERO_IDENTIFICACION($NUMERO_IDENTIFICACION)
    {
        $this->NUMERO_IDENTIFICACION = $NUMERO_IDENTIFICACION;

        return $this;
    }

    /**
     * Get the value of RAZON_SOCIAL
     */

    public function getRAZON_SOCIAL()
    {
        return $this->RAZON_SOCIAL;
    }

    /**
     * Set the value of RAZON_SOCIAL
     *
     * @return  self
     */

    public function setRAZON_SOCIAL($RAZON_SOCIAL)
    {
        $this->RAZON_SOCIAL = $RAZON_SOCIAL;

        return $this;
    }
} ?>



